implementations/rust/ockam/ockam_api/src/cli_state/cli_state.rs

```diff
@@ -82,6 +87,15 @@ impl CliState {
     pub fn set_node_name(&mut self, node_name: impl AsRef<str>) {
         self.database.set_node_name(node_name.as_ref());
     }
+
+    pub fn subscribe(&self) -> Receiver<Notification> {
+        self.notifications.subscribe()
+    }
+
+    // 00: make the notification arg a ref
```

Comment:
I think `// 00:` is meant to be a `// TODO:` ... I use this in a vscode extension that I have :)